### PR TITLE
CTCTRADERS-1902: Added missing param's and rules to header

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

**Review by MatthewAlexanderBeck — COMMENTED** [review]
test this please

**Comment by MatthewAlexanderBeck** [comment]
test this please

File: app/services/DeclarationRequestService.scala [hunk 1]
```diff
@@ -344,20 +372,20 @@ class DeclarationRequestService @Inject()(
...
         typOfDecHEA24       = movementDetails.declarationType.code,
         couOfDesCodHEA30    = Some(routeDetails.destinationCountry.code),
-        agrLocOfGooCodHEA38 = None, // prelodge
-        agrLocOfGooHEA39    = agreedLocationOfGoods(movementDetails, goodsSummary.goodSummaryDetails), // prelodge
-        autLocOfGooCodHEA41 = None,
-        plaOfLoaCodHEA46    = None,
+        agrLocOfGooCodHEA38 = None, // Not required
+        agrLocOfGooHEA39    = agreedLocationOfGoods(movementDetails, goodsSummary.goodSummaryDetails),
+        autLocOfGooCodHEA41 = authorisedLocationOfGoods(goodsSummary.goodSummaryDetails),
+        plaOfLoaCodHEA46    = None, // Journey is currently missing for this
```
Comment (ashwinirudrappa) [inline app/services/DeclarationRequestService.scala:378]: @MatthewAlexanderBeck Do we have any ticket that covers this case " // Journey is currently missing for this"

**Review by ashwinirudrappa — APPROVED** [review]

File: app/services/DeclarationRequestService.scala (hunk 1 above)
Comment (MatthewAlexanderBeck) [reply app/services/DeclarationRequestService.scala:378]: @ashwinirudrappa yes we do. Luke has created a ticket to add this into the journey.

**Pull request merged**